.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/windows/SalesforceMobileSDK-Windows"]
```

Comment:
The plugin repo is flat. Instead of using submodules, we use ```update.sh``` to pull from the source repos. Maybe we should add an entry there for ```Windows```, to keep this repo lean.